source/testing-your-html/index.html.md.erb

```diff
@@ -0,0 +1,53 @@
+---
+title: Test your HTML matches GOV.UK Frontend
+weight: 61
+---
+
+# Test your HTML matches GOV.UK Frontend
+
+You can use our test fixtures to check you're outputting the same HTML that GOV.UK Frontend uses, when you:
+
+- create your own component macros (also called ‘partials’ or ‘templates’)
+- update GOV.UK Frontend
+
+## Using the HTML test files
+
+If you [installed GOV.UK Frontend with Node.js package manager (npm)](/installing-with-npm), you can find the `fixtures.json` file for each component in the `node_modules/govuk-frontend/govuk/components/COMPONENT-NAME/` folder, where `COMPONENT-NAME` is the name of the component.
+
+<%= warning_text('You should not test your HTML using the YAML files in the <code>src/</code> folder.') %>
```

Comment:
Still not 100% sure that this is terribly important to mention – or at least that it's important enough to warrant being in a callout. If people do it nothing bad's going to happen, it's just that we might change those files without giving them any warning.

Reply:
No probs - happy to remove this.

Reply:
Removed in eaeabe9